boards/arm/nucleo_f303re/nucleo_f303re.dts

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2020 Paul M. Bendixen
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <st/f3/stm32f303Xe.dtsi>
+#include "arduino_r3_connector.dtsi"
+
+/ {
+	model = "STMicroelectronics STM32F303RE-NUCLEO board";
+	compatible = "st,stm32f303re-nucleo", "st,stm32f303";
+
+	chosen {
+		zephyr,console = &usart2;
+		zephyr,shell-uart = &usart2;
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		green_led_2: led_2 {
+			gpios = <&gpioa 5 GPIO_INT_ACTIVE_HIGH>;
```

Comment:
@paulbendixen You need to rebase on master and fix the GPIO device tree flags.

Reply:
Thanks @henrikbrixandersen !